### PR TITLE
GlobalDNS docs

### PR DESCRIPTION
**Pull request by prachidamle**
Here are the docs for GlobalDNS feature. https://github.com/rancher/docs/issues/1014

### CODE REVIEW

**Comment by deniseschannon** [comment]
@prachidamle can you rebase?

File: content/rancher/v2.x/en/tools/globaldns/_index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+Rancher's GlobalDNS feature provides a way to program an external DNS provider for the kubernetes workloads. It is programmed at a global level in Rancher, and can be used to program DNS for workloads that may be running in multiple kubernetes clusters. 
+
+This feature helps in achieving high availability for an application that may be running on multiple kubernetes clusters, by programming an FQDN for your application against IP addresses from different clusters where the application is running. 
+
+
+## External DNS Provider Support
+
+Rancher deploys the provider implementations available under [kubernetes-incubator/external-dns](https://github.com/kubernetes-incubator/external-dns) project.
+The External DNS providers supported in Rancher 2.2 release are AWS Route53, CloudFlare and alidns. Support for more providers can be added in future releases.
```
Comment (alena1108) [inline content/rancher/v2.x/en/tools/globaldns/_index.md]: nit: shall we make alidns upper/camelcase? We should also mention that Route53 is the default provider.

File: content/rancher/v2.x/en/tools/globaldns/_index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+1. Enter the **FQDN** you wish to program on the external DNS.
+1. Assign a Global DNS Provider to this entry.
+1. Choose whether this DNS entry will be programmed for a Multi Cluster App or for workloads under some Cluster Projects.
+1. Configure the **DNS TTL** value in seconds - by default it will be 300 seconds.
+1. Under **Member Access**, add any users you may want to add as "Owner" of this entry. They can edit/delete the Global DNS entry.
+
+
+## Add Annotation to an Ingress to program the External DNS
+
+To program an external DNS provider like route53 or cloudflare for your workloads, please follow the steps below.
```
Comment (alena1108) [inline content/rancher/v2.x/en/tools/globaldns/_index.md]: Nit: we should use upper case for provider names, just like we did earlier

File: content/rancher/v2.x/en/tools/globaldns/_index.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+1. Click **Create**.
+{{% /accordion %}}
+
+
+### Add a Global DNS Entry
+
+1. From the **Global View**, select **Tools > Global DNS Entries**
+1. Enter the **FQDN** you wish to program on the external DNS.
+1. Assign a Global DNS Provider to this entry.
+1. Choose whether this DNS entry will be programmed for a Multi Cluster App or for workloads under some Cluster Projects.
```
Comment (alena1108) [inline content/rancher/v2.x/en/tools/globaldns/_index.md]: I think we should elaborate a bit more on Multi cluster app and workloads. How the are really meant to define the scope in which we search for ingress having a specific annotation.

**Comment by alena1108** [comment]
@prachidamle we should note down that Global DNS is only available in HA setups with a local cluster enabled.

**Comment by alena1108** [comment]
LGTM

File: content/rancher/v2.x/en/tools/globaldns/_index.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+---
+title: GlobalDNS
+aliases:
+---
+
+Rancher's GlobalDNS feature provides a way to program an external DNS provider for the kubernetes workloads. It is programmed at a global level in Rancher, and can be used to program DNS for workloads that may be running in multiple kubernetes clusters. 
+
+This feature helps in achieving high availability for an application that may be running on multiple kubernetes clusters, by programming an FQDN for your application against IP addresses from different clusters where the application is running.
+
+Note that, Global DNS feature is only available in HA setups with a local cluster enabled.
```
Comment (ankur111) [inline content/rancher/v2.x/en/tools/globaldns/_index.md]: please list full forms of all acronyms (FQDN) at least once
what is "local cluster"?

File: content/rancher/v2.x/en/tools/globaldns/_index.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+
+## External DNS Provider Support
+
+Rancher deploys the provider implementations available under [kubernetes-incubator/external-dns](https://github.com/kubernetes-incubator/external-dns) project.
+The External DNS providers supported in Rancher 2.2 release are AWS Route53, CloudFlare and AliDNS.  AWS Route53 is the default provider. Support for more providers can be added in future releases.
```
Comment (ankur111) [inline content/rancher/v2.x/en/tools/globaldns/_index.md]: may be added in future releases

File: content/rancher/v2.x/en/tools/globaldns/_index.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+## Global DNS Workflow
+
+### Add a Global DNS Provider
+
+1. From the **Global View**, select **Tools > Global DNS Providers**
+1. To add a provider, choose from the available provider options and configure the GlobalDNSProvider with necessary credentials and an optional domain
+1. You can also assign other users as members to this provider. Members added get 'Owner' role and can manage the globalDNSProvider.
```
Comment (ankur111) [inline content/rancher/v2.x/en/tools/globaldns/_index.md]: Global DNS Provider OR GlobalDNSProvider or globalDNSProvider?

File: content/rancher/v2.x/en/tools/globaldns/_index.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+1. Click **Create**.
+{{% /accordion %}}
+
+
+### Add a Global DNS Entry
+
+1. From the **Global View**, select **Tools > Global DNS Entries**
+1. Enter the **FQDN** you wish to program on the external DNS.
+1. Assign a Global DNS Provider to this entry.
+1. Choose whether this DNS entry will be programmed for a Multi Cluster App or for workloads under some Cluster Projects. This defines the scope in which Rancher will search for an ingress having a specific annotation, to be programmed on to the external DNS. Thus when the targets of the DNS entry are set, then ingresses in some other projects or part of another MultiClusterApp will not be programmed on the external DNS.
```
Comment (ankur111) [inline content/rancher/v2.x/en/tools/globaldns/_index.md]: Multi cluster app OR Multi-cluster app OR MultiClusterApp?

File: content/rancher/v2.x/en/tools/globaldns/_index.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+1. Configure the **DNS TTL** value in seconds - by default it will be 300 seconds.
+1. Under **Member Access**, add any users you may want to add as "Owner" of this entry. They can edit/delete the Global DNS entry.
```
Comment (ankur111) [inline content/rancher/v2.x/en/tools/globaldns/_index.md]: GlobalDNS or Global DNS?

**Comment by prachidamle** [comment]
@ankur111 please re-review

**Pull request merged**